stubs/django-import-export/import_export/formats/base_formats.pyi

```diff
@@ -1,9 +1,12 @@
 from _typeshed import Incomplete, ReadableBuffer
+from logging import Logger
 from typing import IO, Any, ClassVar
 from typing_extensions import Self, TypeAlias
 
 Dataset: TypeAlias = Incomplete  # tablib.Dataset
```

Comment:
@srittau, as per https://github.com/python/typeshed/pull/12990/files#r1921560447, do you think we can do the same here?

Reply:
Sure, let's try it!

Reply:
Seems like it doesn't work with pyright, as we make use of the imported symbol in annotations